utils/projects.json

```diff
@@ -22,10 +22,10 @@
            "https://build.opnfv.org/ci"
         ],
         "gitlab:issue": [
-            "https://gitlab.com/gitlab-org/gitlab-ce"
+            "https://gitlab.com/gitlab-org/gitlab-development-kit.git"
```

Comment:
Can you please remove the `.git` from the project URL? 
I think it would cause an error, not sure though. But, in any case, a clean project URL would look good.

```
        "gitlab:issue": [
-            "https://gitlab.com/gitlab-org/gitlab-development-kit.git"
+            "https://gitlab.com/gitlab-org/gitlab-development-kit"
        ],
        "gitlab:merge": [
-            "https://gitlab.com/gitlab-org/gitlab-development-kit.git"
+            "https://gitlab.com/gitlab-org/gitlab-development-kit"
        ]

Reply:
Oops, sorry about that!  Should be good now